jenkins-agent.yaml

```
@@ -27,9 +27,21 @@ spec:
       command:
         - cat
       tty: true
+    - name: curl-jq
+      image: peterevans/curl-jq
```

This image hasn't been updated in 2 years and seems to be unmaintained. 

Might be better off using the official curl image and running `apk add --no-cache jq` in the pipeline
```suggestion
      image: curlimages/curl
```